recipes/soapysdr-module-airspyhf/bld.bat

```diff
@@ -0,0 +1,25 @@
+setlocal EnableDelayedExpansion
+@echo on
+
+:: Make a build folder and change to it
+mkdir build
+if errorlevel 1 exit 1
+cd build
+if errorlevel 1 exit 1
+
+:: configure
+cmake -G "Ninja" ^
+    -DCMAKE_BUILD_TYPE:STRING=Release ^
+    -DCMAKE_INSTALL_LIBDIR:STRING=lib ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_PREFIX_PATH:PATH="%LIBRARY_PREFIX%" ^
+    ..
+if errorlevel 1 exit 1
+
+:: build
+cmake --build . --config Release -- -j%CPU_COUNT%
+if errorlevel 1 exit 1
+
```

Comment:
Is there a `ctest` or `make check` we can run here?

Reply:
No, I don't see any tests in the source code. Our own functionality checks are about all that we can do, and even those are very simple.